lib/api/fields/types/views/radio.php

```diff
@@ -26,10 +26,13 @@
 		?>
 			<span class="screen-reader-text"><?php echo esc_html( $image['screen_reader_text'] ); ?></span>
 			<img src="<?php echo esc_url( $image['src'] ); ?>" alt="<?php echo esc_html( $image['alt'] ); ?>" />
+			<input id="radio_<?php echo $clean_value; ?>" class="screen-reader-text" type="radio" name="<?php echo esc_attr( $field['name'] ); ?>" value="<?php echo $clean_value; ?>"<?php checked( $value, $field['value'], 1 ); echo beans_esc_attributes( $field['attributes'] ); ?> /> <?php // @codingStandardsIgnoreLine - WordPress.XSS.EscapeOutput.OutputNotEscaped - Escaped above. ?>
```

Comment:
`// @codingStandards...` is deprecated, and never supported this format of disabling individual sniffs / checks anyway. Use `// phpcs:ignore ...`.

Reply:
@GaryJones The minimum PHP version for Beans 1.x is PHP 5.2.  At that version, the only option we have is to use `@codingStandards`.  In Beans 2.0, we are kicking up to PHP 5.6.  At that point, then we will switch the ignores to the new syntax in order to ignore individual sniffs.